### ISSUE TITLE
"Unblocking is unsupported on newer OS versions"

### ISSUE THREAD

**Issue by mrsorensen** [parkerlreed/adb-block#5]
Already blocked a couple of apps. Any fix?

**Comment by parkerlreed** [comment]
Short of factory reset, I haven't found a way.

Comment over on XDA

````
This pm uninstall -k --user 0 com.my.demo.app only removes the app from user 0 ( current user ) 
Sooo to reinstall and app, you will need to factory default your device
````

**Comment by parkerlreed** [comment]
This has been updated! Fixed in 28456485be2a0cba89468e6e5d604fe256533c71
